game.html

```diff
@@ -3,21 +3,29 @@
   var answer = Number.parseInt('18');
 
```

Comment:
Keep in mind that you want your code readable. Too much white space can be tough to read for other programmers, and will ultimately extend your code into more lines than you want.  Try thinking of your code blocks as paragraphs, and use white space in places that helps delineate separations in those blocks. 

This line break here is unnecessary.